docs/modules/debug/network-analyzer.md

```diff
@@ -0,0 +1,36 @@
+<%( TOC )%>
+
+# Network Analyzer window
+
+The Network Analyzer window is an Unity Editor window which displays live bandwidth usage for a worker broken down by component type. This window can assist you in finding networking bottlenecks in your game.
```

Comment:
```suggestion
The Network Analyzer window is a Unity Editor window which displays live bandwidth usage for a worker broken down by component type. This window can assist you in finding networking bottlenecks in your game.
```

`a Unity` feels more natural but `an Unity` is what I've been taught as a kid 🎃

Reply:
Yeah I had the exact same thought when writing this.

Reply:
@ElleEss has decreed it shall be `a Unity`